lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -63,6 +66,36 @@ def ajax_content_response(request, course_key, content):
     })
 
 
+def track_forum_event(request, event_name, course, obj, data):
+    """
+    Send out an analytics event when a forum event happens. Works for threads,
+    responses to threads, and comments on those responses.
+    """
+    user = request.user
+    data['id'] = obj.id
+    id_map = get_discussion_id_map(course)
```

Comment:
regarding `category` / nomenclature i don't have a strong feeling about what the name should be, just want to make sure we don't add anything inconsistent on the fly.